### ISSUE TITLE
<div class="xdsoft_datetimepicker"> created on the main page.

### ISSUE THREAD

**Issue by ghost** [xdan/datetimepicker#120]
Hello.
In my project there is one main page and other pages are loaded AJAX requests.
For example, index.html is main page with <div id="scheduleContent">. And schedule.html is page that loaded to index.html <div id="scheduleContent"> with AJAX request. Page schedule.html contains many datetimepickers.
Whenever I load schedule.html page to index.html, at the end of the index.html page  <div class="xdsoft_datetimepicker"> elements are created for each datetimepicker on schedule.html page.
When I move to another page (for example, info.html that AJAX loaded too) , the index.html page, of course, contains xdsoft_datetimepicker elements. This elements not deleted, because it created on the end of index.html. This elements not deleted, because it created on the end of index.html.
Many times when I load the schedule.html page new elements created and not deleted afterwards. Therefore, the application's memory consumption is growing.

**Comment by degrootconsultancy** [comment]
I agree with @ivKoronaLiftProg4, that html output should be near the element so it automatically gets destroyed if the container div of that element is destroyed.
